hs_core/templates/pages/genericresource.html

```diff
@@ -1424,6 +1471,36 @@ <h4 class="modal-title">Confirm files deletion</h4>
             if (task_id)
                 update_download_status(task_id, download_path);
 
+            $('.contact-table .sortable').sortable({
+                axis: "y",
+                stop: function( event, ui ) {
+                    var forms = $("#authors-table .drag-indicator > form");
+
+                    // Set the new order value in the form items
+                    for (var i = 0; i < forms.length; i++) {
+                        $(forms[i]).find("input.order-input").attr("value", i + 1);
+                        $("#id_creator-" + i + "-order").attr("value", $("input[name='creator-" + i + "-order']").val());
+                    }
+
+                    $form = $(ui.item.find(".drag-indicator > form"));
+                    var url = $form.attr('action');
+                    var datastring = $form.serialize();
+                    $("html").css("cursor", "progress");
+
+                    $.ajax({
+                        type: "POST",
+                        url: url,
+                        data: datastring,
+                        success: function (result) {
+                            $("html").css("cursor", "initial");
+                        },
+                        error: function (xhr, errmsg, err) {
+                            $("html").css("cursor", "initial");
```

Comment:
It'd be better to print the error message to console if ajax response returned with error message.